content/docs/started/getting-started-minikube.md

```diff
@@ -9,7 +9,7 @@ of Kubeflow running on top of Minikube. Minikube runs a simple, single-node
 Kubernetes cluster inside a virtual machine (VM).
 
 By the end of this document, you'll have a local installation of Minikube kubernetes cluster along with all the default core components of
-Kubeflow deployed as services in the pods. You should be able to access JupyterHub notebooks, and the Kubeflow Dashboard.
+Kubeflow deployed as services in the pods. You should be able to access Jupyter notebooks, and the Kubeflow Dashboard.
```

Comment:
Same comment regarding Jupyter Notebook as an application. Consider changing:

You should be able to access Jupyter notebooks

to:

You should be able to access Jupyter Notebook servers

Reply:
See my previous replies. I think most people want to know whether they can access Jupyter notebooks in Kubeflow. The fact that they need to set up a notebook server first is an implementation detail.